apps/src/aichat/views/AichatView.tsx

```diff
@@ -163,6 +165,13 @@ const AichatView: React.FunctionComponent = () => {
             headerContent={chatWorkspaceHeader}
             rightHeaderContent={renderChatWorkspaceHeaderRight(() => {
               dispatch(clearChatMessages());
+              analyticsReporter.sendEvent(
+                EVENTS.CHAT_ACTION,
+                {
```

Comment:
Nit: is payload required? Seems potentially redundant?

Reply:
There are 2 possible chat actions - 'Clear chat history' and 'Clear chat history'. In the [spec](https://docs.google.com/document/d/1yIez6ooHbfwfM2W16pQvGHgKTe97AdGJN7YECTOpmBU/edit#heading=h.ghfocbs7b58g) @samantha-code wrote that she would like to know which action was taken. 
Lmk if I'm interpreting your question correctly - thanks!

Reply:
Ah sorry, I didn't realize it was multiple things happening (clear v copy) within the same event name. nvm then, this seems good!